src/spec-node/devContainersSpecCLI.ts

```diff
@@ -328,18 +328,24 @@ async function doBuild({
 			throw new ContainerError({ description: `Dev container config (${uriToFsPath(configFile || getDefaultDevContainerConfigPath(cliHost, workspace!.configFolderPath), cliHost.platform)}) not found.` });
 		}
 		const { config } = configs;
-		let imageNameResult = '';
+		let imageNameResult = [''];
+		
+		// Support multiple use of `--image-name` 
+		const imageNames = (argImageName && (Array.isArray(argImageName) ? argImageName : [argImageName]) as string[]) || undefined;
 
 		if (isDockerFileConfig(config)) {
 
 			// Build the base image and extend with features etc.
-			const { updatedImageName } = await buildNamedImageAndExtend(params, config, argImageName);
-
-			if (argImageName) {
-				if (!buildxPush) {
-					await dockerPtyCLI(params, 'tag', updatedImageName, argImageName);
+			let { updatedImageName } = await buildNamedImageAndExtend(params, config, imageNames);
+			updatedImageName = (updatedImageName && (Array.isArray(updatedImageName) ? updatedImageName : [updatedImageName]) as string[]) || [''];
```

Comment:
This line shouldn't be necessary, shouldn't it always be an array? (The handling of argImageName above is an exception due to the typing being incomplete.)

Reply:
Makes sense. Modified.